functional-tests/src/test/resources/suites/simpleTestSuite.xml

```diff
@@ -23,11 +23,11 @@
             <!--<class name="org.eclipse.che.selenium.dashboard.CreateWorkspaceTest"/>-->
             <!--<class name="org.eclipse.che.selenium.dashboard.CreateAndDeleteProjectsTest"/>-->
             <!--<class name="org.eclipse.che.selenium.dashboard.RenameWorkspaceTest"/>-->
-            <class name="org.eclipse.che.selenium.miscellaneous.DialogAboutTest"/>
-            <class name="com.redhat.che.functional.tests.WorkspaceTest"/>
+<!--             <class name="org.eclipse.che.selenium.miscellaneous.DialogAboutTest"/> -->
```

Comment:
Why do not you want to delete those commented lines ?

Reply:
This class should not be there, thank you for your notice. But commands will still remain there, because these commented classes are possible candidates to be added to our suite. We have them enabled but it seemed like some of them were not stable. We decided to temporary disable them to ensure that our tests are stable enough (there were not enough time to investigate and agree on adding them again or removing them).